.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8','3.9','3.10']
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"        
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies
+        shell: bash
+        run: |
+          python -m pip install --upgrade pip
+          pip install Bottleneck
```

Comment:
Yeah it is too bad this one dependency has to specifically be installed like this, so maybe adding to `setup.py` would be better?

Reply:
Oh I see this is done in #102 ,so nevermind.

Reply:
Yes, I thought it best just to include this in the `setup.py`. So now I can take out this line at least!